pkg/distro/generic/images.go

```diff
@@ -83,6 +83,12 @@ func osCustomizations(t *imageType, osPackageSet rpmmd.PackageSet, options distr
 
 		osc.Users = users.UsersFromBP(c.GetUsers())
 		osc.Users = append(osc.Users, imageConfig.Users...)
+
+		// we also don't really want to lock the root account (as far
+		// as it exists) on installer ISOs
+		if imageConfig.LockRoot != nil {
+			osc.LockRoot = *imageConfig.LockRoot
+		}
```

Comment:
Please use `common.ValueOrEmpty` for this, is much shorter.

Reply:
I didn't know it existed, it's not used at all in this file yet. I'll probably do a follow-up to rewrite the idiom everywhere as it is indeed nicer.

Reply:
Tried my best with `grep` but there is probably more: https://github.com/osbuild/images/pull/2000